tests/integ/test_workflow.py

```diff
@@ -986,6 +987,112 @@ def test_two_step_lambda_pipeline_with_output_reference(
             pass
 
 
+def test_one_step_emr_pipeline(sagemaker_session, role, pipeline_name, region_name):
```

Comment:
also, we need to somehow setup a cluster to run the test. otherwise it will fail.